src/features/artifacts/components/ArtifactList.tsx

```diff
@@ -17,9 +17,10 @@ export interface IArtifactsProps {
   artifacts: Artifact[];
 }
 
-export const ArtifactList = ({ artifacts }: IArtifactsProps) => {
+export const ArtifactsList = ({ artifacts }: IArtifactsProps) => {
```

Comment:
Can you please change this back to `ArtifactList`?

Reply:
Sure, no problem